contracts/leaser/src/migrate.rs

```diff
@@ -2,101 +2,305 @@ use lease::api::MigrateMsg;
+
+        {
+            let exp = MigrationResult {
+                next_customer: Some(customer_addr1),
+                ..Default::default()
+            };
+            assert_eq!(
+                Ok(exp),
+                super::migrate_leases(test_customers(), new_code, 0)
+            );
+        }
+        {
+            let mut exp = add_expected(MigrationResult::default(), &lease1, new_code);
+            exp.next_customer = Some(customer_addr2.clone());
+            assert_eq!(
+                Ok(exp),
+                super::migrate_leases(test_customers(), new_code, 1)
+            );
+        }
+        {
+            let mut exp = add_expected(MigrationResult::default(), &lease1, new_code);
+            exp.next_customer = Some(customer_addr2);
+            assert_eq!(
+                Ok(exp),
+                super::migrate_leases(test_customers(), new_code, 2)
+            );
+        }
+        {
+            let exp = add_expected(MigrationResult::default(), &lease1, new_code);
+            let exp = add_expected(exp, &lease21, new_code);
+            let mut exp = add_expected(exp, &lease22, new_code);
+            exp.next_customer = Some(customer_addr3);
+            assert_eq!(
+                Ok(exp),
+                super::migrate_leases(test_customers(), new_code, 3)
+            );
+        }
+        {
+            let exp = add_expected(MigrationResult::default(), &lease1, new_code);
+            let exp = add_expected(exp, &lease21, new_code);
+            let exp = add_expected(exp, &lease22, new_code);
+            let mut exp = add_expected(exp, &lease3, new_code);
+            exp.next_customer = Some(customer_addr4.clone());
+            assert_eq!(
+                Ok(exp),
+                super::migrate_leases(test_customers(), new_code, 4)
+            );
+        }
+        {
+            let exp = add_expected(MigrationResult::default(), &lease1, new_code);
+            let exp = add_expected(exp, &lease21, new_code);
+            let exp = add_expected(exp, &lease22, new_code);
+            let mut exp = add_expected(exp, &lease3, new_code);
+            exp.next_customer = Some(customer_addr4);
+            assert_eq!(
+                Ok(exp),
+                super::migrate_leases(test_customers(), new_code, 5)
+            );
+        }
+        {
+            let exp = add_expected(MigrationResult::default(), &lease1, new_code);
+            let exp = add_expected(exp, &lease21, new_code);
+            let exp = add_expected(exp, &lease22, new_code);
+            let exp = add_expected(exp, &lease3, new_code);
+            let exp = add_expected(exp, &lease41, new_code);
+            let exp = add_expected(exp, &lease42, new_code);
+            let mut exp = add_expected(exp, &lease43, new_code);
+            exp.next_customer = None;
+            assert_eq!(
+                Ok(exp),
+                super::migrate_leases(test_customers(), new_code, 7)
+            );
```

Comment:
Expected expression is supposed to be the second argument to the macro.
```suggestion
            assert_eq!(
                super::migrate_leases(test_customers(), new_code, 7),
                Ok(exp)
            );
```

Reply:
same

Reply:
same